src/vivarium_public_health/risks/base_risk.py

```diff
@@ -270,19 +283,19 @@ def get_exposure_pipeline(self, builder: Builder) -> Pipeline:
     ########################
 
     def on_initialize_simulants(self, pop_data: SimulantData) -> None:
-        propensity_values = self.randomness.get_draw(pop_data.index)
-        df = pd.DataFrame(
-            {
-                self.propensity_column_name: self.randomness.get_draw(pop_data.index),
-                self.exposure_column_name: self.exposure_distribution.ppf(propensity_values),
-            }
+        propensity = pd.Series(
+            self.randomness.get_draw(pop_data.index), name=self.propensity_column_name
         )
-        self.population_view.update(df)
+        self.population_view.update(propensity)
+        self.update_exposure_column(pop_data.index)
 
     def on_time_step_prepare(self, event: Event) -> None:
-        exposure_values = self.exposure(event.index)
```

Comment:
This is identical to how we updated exposures `on_initialize_simulants`, so refactor into a new `update_exposure_column()` method